### ISSUE TITLE
cannot update available font list on win64

### ISSUE THREAD

**Issue by rvillemeur** [guillep/pharo-core#10949]
**Bug description**
In the settings browser, one can, on clicking on the name of the font, display a file dialog to select another font, or update the font list with font available on the system.

**To Reproduce**
1. Go to 'setting browser'
2. Click on 'standard fonts - default - name on the font' to open the font selector dialog box
3. click on update.
4. no font are loaded

**Expected behavior**
update the list of the font available on the system, so one can select another font type

**Version information:**
 - OS: Windows 64 bit
 - Version: 11
 - Pharo Version 10

**Expected development cost**
Easy. The issue is coming from the method ```FreeTypeFontProvider >>platformAbsoluteDirectories```, which does not correctly detect windows platform when using 64 bit windows.

The code could be:
```smalltalk
platformAbsoluteDirectories
	Smalltalk os isWindows ifTrue: [ ^ self winFontDirectories ].
	Smalltalk os isUnix ifTrue: [ ^ self unixFontDirectories ].
	Smalltalk os isMacOSX ifTrue: [ ^ self macOSXFolderDirectories ].
	^ {  }
```

**Additional context**

**Comment by tesonep** [comment]
I think the solution is correct, would you like to make a PR with the proposed changes?
Thanks.

**Comment by rvillemeur** [comment]
fix with commit 9b3f6689e35e00baaaf1c6b5399eb1f103f5b309
